package.json

```diff
@@ -44,6 +44,7 @@
     "./src/zindex": "./src/props.zindex.js",
     "./src/masks.edges": "./src/props.masks.edges.js",
     "./src/masks.corner-cuts": "./src/props.masks.corner-cuts.js",
+    "./src/brand": "./src/props.brand.js",
```

Comment:
add another entry into this file that points to the css file, somewhere next to the other colors 

something like "./brand": "path-to-brand-css-file",